lib/ain-grpc/src/rpc.rs

```diff
@@ -228,6 +225,16 @@ impl MetachainRPCServer for MetachainRPCModule {
             })
     }
 
+    fn get_pending_transaction(&self) -> RpcResult<Vec<EthPendingTransactionInfo>> {
+        get_pool_transactions()
```

Comment:
For consistency: https://github.com/DeFiCh/ain/blob/0266fbf5f004534ef25b5441030dfcec5e19f0fa/lib/ain-grpc/src/rpc.rs#L213

```suggestion
        ain_cpp_imports::get_pool_transactions()
```